cibersort/macros.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<macros>
+
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="3.6.2">R</requirement>
+            <requirement type="package">reshape2</requirement>
+            <requirement type="package">ggplot2</requirement>
+            <requirement type="package">dplyr</requirement>
+            <requirement type="package">e1071</requirement>
+            <requirement type="package">parallel</requirement>
+            <requirement type="package">preprocess</requirement>
+            <requirement type="package" version="1.04">CIBERSORT</requirement>
+        </requirements>
+    </xml>
+
+    <xml name="version_cmd">
+        <version_command>echo @VERSION@</version_command>
+    </xml>
+    <xml name="citations">
+        <citations>
+        </citations>
+    </xml>
+
+    <token name="@VERSION@">1.04</token>
+    <token name="@CMD_BEGIN@">Rscript ${__tool_directory__}/run_cibersort.R</token>
```

Comment:
This token is not be used. Also, is pointing to a script that should no longer be used.

Reply:
`@CMD_BEGIN@` removed from the macros file